snowflake/pkg/infra/manager.go

```diff
@@ -37,6 +36,8 @@ import (
 
 	"antrea.io/theia/snowflake/database"
 	sf "antrea.io/theia/snowflake/pkg/snowflake"
+	utils "antrea.io/theia/snowflake/pkg/utils"
```

Comment:
nit: renaming the package doesn't seem necessary here

Reply:
Thanks, addressed.